lms/static/js/learner_dashboard/models/course_card_model.js

```diff
@@ -164,7 +164,7 @@
                         } else if (end && endDate < now) {
                             dateString += ' - Ended ' + end;
                         }
-                    } else if (pacingType === 'instructor_paced') {
+                    } else {
```

Comment:
because pacingType can be null, so in that case we just show dates normally without the self paced text